docs/consistency-options.md

```diff
@@ -0,0 +1,69 @@
+# Consistency Options
+
+SpiceDB's [V1 API] provides the ability to specify the *consistency* level at which API calls are made.
+
+Consistency is provided via the [Consistency block] on API calls.
+
+[V1 API]: https://buf.build/authzed/api/tree/main/authzed/api/v1
+[Consistency block]: https://buf.build/authzed/api/file/main/authzed/api/v1/permission_service.proto#L50
```

Comment:
That URL is guaranteed to change, you probably meant this one: https://buf.build/authzed/api/docs/main/authzed.api.v1#authzed.api.v1.Consistency